frontend/src/components/associations/page/edit/AssociationEditPage.tsx

```diff
@@ -44,7 +45,9 @@ export const AssociationEditPage = ({
             ? "Détails : " +
               (error.response.status === 403
                 ? "vous n’avez pas le droit de modifier cette page."
-                : error.response.data.detail)
+                : error.response.status === 400 && error.response.data.title
```

Comment:
I'm afraid we always get the serialized error

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/4300678/87573668-8cbccc00-c6cd-11ea-8a07-ae83f0a5c9f7.png">